app/routes/session.py

```diff
@@ -132,7 +132,16 @@ def get_sign_out():
     """
     Signs the user out of eQ and redirects to the log out url.
     """
-    log_out_url = get_survey_config().account_service_log_out_url
+    todo = request.args.get("todo", "") == "True"
```

Comment:
This needs updating also after you updated template
` todo = request.args.get("todo", "") == "true"`

Reply:
This might need to stay as `"True"` since that's how it is formatted when returned from the template.

Reply:
I think we should either check where case don't matter (what most type enforced APIs does), or just check the existence of the key, which is what we do for resume, given we don't have an alternative path, it just if it has to do or not.

Reply:
Re-introduced the case fix and default fallback `false`, thanks.

Reply:
Taking onboard what Mebin has said and doing a slight refactor, would this be better? 

```
    survey_config = get_survey_config()
    log_out_url = (
        survey_config.account_service_todo_url
        if "todo" in request.args
        else survey_config.account_service_log_out_url
    )

    # Check for GET as we don't want to log out for HEAD requests
    if request.method == "GET":
        logout_user()

    return redirect(log_out_url)